src/routes/docs/advanced/platform/release-policy/+page.markdoc

```diff
@@ -0,0 +1,161 @@
+---
+layout: article
+title: Release policy
+description: Understand how Appwrite releases and versions its platforms and APIs.
+---
+We value the trust of developers in Appwrite as the backbone of their applications. 
+Our release policy is designed to provide developers with a reliable and consistent experience when using Appwrite. 
+We are committed to providing support for our API, SDKs, and product versions for a reasonable length of time, 
+and we follow industry-standard versioning protocols. 
+Appwrite will prioritize security updates and will release new versions immediately to fix any security vulnerabilities.
+
+# Schedule {% #schedule %}
+
+We work to release a new minor version of the product every quarter, which will include new features and enhancements. 
+
+We prioritize the timely release of patch versions with bug fixes and security updates on top of these feature releases, 
+and we make every effort to ensure that our releases are thoroughly tested and stable before they are made available to developers. 
+
+In rare cases where there are significant delays or changes to our release schedule, we will notify through our website, Discord server, 
```

Comment:
it would be good to link website, discord, newsletter, etc to relevant pages.

Reply:
Done.